django/views/debug.py

```diff
@@ -893,6 +893,28 @@ def default_urlconf(request):
   <h2>Request information</h2>
 
 {% if request %}
+  <h3 id="user-info">USER</h3>
+  {% if request.user %}
+    <table class="req">
+      <thead>
+        <tr>
+          <th>Variable</th>
+          <th>Value</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for var in request.user.items %}
```

Comment:
I think the template system silences the exception -- I just observed that no data appears in the table.

Your proposal is what I had in mind.